scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -325,10 +264,16 @@ nomatch.df <- complete.df %>%
 ```
 ## Mutations matching variants of concern
 
-The following tables list the called SNVs found by [*LoFreq*](https://csb5.github.io/lofreq/) and translated 
-to protein mutations by [*Ensemble VEP*](https://covid-19.ensembl.org/info/docs/tools/vep/index.html). Table 1 shows 
-the mutations matching signature mutations for given SARS-CoV-2 variants. 
-Table 2 shows all other mutations without any match. 
+The following tables list the called SNVs. Table 1 shows the mutations matching signature mutations for given SARS-CoV-2 
+variants. Table 2 shows all other mutations without any match. The columns are described as follows:  
+1. **SNV** - nucleotide mutation in the pattern of: reference base- position nucleotide sequence - mutation base 
+2. annotated gene  
+3. translated amino acid (AA) mutation in the pattern of: reference AA - position AA sequence - mutation AA  
```

Comment:
See above.  Use commas.

Reply:
because I think AA is the conventional abbreviation to amino acid